scripts/gas-refund-program/transactions-indexing/swaps-subgraph.ts

```diff
@@ -134,7 +134,7 @@ export async function getSuccessfulSwaps({
       variables,
     });
 
-    const swaps = data.data.swaps;
+    const swaps = data.data?.swaps || [];
```

Comment:
I'm supposing you faced [this error](https://paraswap.slack.com/archives/C01B1KVP2RZ/p1691656788076149) and omitted this way?

I'm afraid there's more to it :(. I think we rather need to decrease [slice size here](https://github.com/paraswap/paraswap-volume-tracker/blob/e29b0bd2faefae0f56b0d93febca55028ea91a02/scripts/gas-refund-program/transactions-indexing/fetchRefundableTransactions.ts#L22), to smth like 4 hours maybe

Reply:
Also while you're at it, could you please fix fetchSwap func? The part that needs changing is pageSize param. Noticed it while debugging the above 

```
  const fetchSwaps = async ({ skip, pageSize }: QueryPaginatedDataParams) => {
    const variables = Object.assign(
      {},
      {
        number_gte: startTimestamp,
        number_lt: endTimestamp,
        skip,
        // apparently thegraph's param for page size is called "first"
        first: pageSize,
      },
```